index.html

```diff
@@ -699,19 +699,19 @@ <h2>Help your friends find some good learning resources</h2>
                     <p>3 Useful Dev Resources</p>
                     <ul>
                         <li>
-                            <a href="#" target="_blank" title="[How to organize a study schedule?](https://www.youtube.com/watch?v=DemagUML778&t=72s)">Resource 1</a>
+                            <a href="https://www.youtube.com/watch?v=DemagUML778&t=72s" target="_blank" title="How to organize a study schedule?">Resource 1</a>
```

Comment:
Instead of naming Resource as "Resource 1" tried to give it a title of the link, for reference you can check out other's contribution